locales/en/service-account.json

```diff
@@ -0,0 +1,12 @@
+{
+  "service_accounts": "Service Accounts",
+  "service_accounts_small": "Service accounts",
+  "create_service_account_to_generate_credentials": "Create a service account to generate credentials. Manage the service accounts you create on the",
+  "create_service_account": "Create service account",
+  "current_instance": "Accounts do not have access to this instance unless granted under the instance’s",
+  "access_tab": "Access tab.",
+  "alter_allow" : "The owner, the organization administrator, or users with the Allow Alter permission on this instance can manage access.",
+  "sasl_plain_description": " Use your service account credentials (Client ID and Client secret) as the user name and password to authenticate your client with the Kafka instance.",
+  "sasl_oauthbearer_description": "Use your service account credentials and a token endpoint URL to authenticate your client with the Kafka instance.",
+  "sasl_oauthbearer_popover_content": "OpenShift Streams for Apache Kafka uses the industry-standard OAuth protocol via the SASL/OAUTHBEARER provided by Apache Kafka."
```

Comment:
The three strings above are not related to service accounts:
`sasl_plain_description`
`sasl_oauthbearer_description`
`sasl_oauthbearer_popover_content`

Should these be moved to `"connection_tab": { ... }` in kafka.json instead?